detection_rules/integrations.py

```diff
@@ -26,6 +26,7 @@
 from .schemas import definitions
 
 MANIFEST_FILE_PATH = get_etc_path('integration-manifests.json.gz')
+NUM_LATEST_RULE_VERSIONS = 2
```

Comment:
Thank you for putting together this PR so quickly @Mikaayenson 🙏 

If we compare the currently published latest version of the package VS a package that would be produced after this optimization:

- what would be the size of each in megabytes?
- what would be the number of assets in each?

Reply:
Oh, I just saw in the collapsed "Debugging" section that it's 5553 assets VS 2199 assets after the optimization.

I guess this makes the size in bytes lower by ~50% as well.